README.md

```diff
@@ -43,17 +51,31 @@ Follow these steps to get started with the React components.
 
 ### 1. Install the packages
 
+```sh
+npm i @digdir/designsystemet-react @digdir/designsystemet-tokens @digdir/designsystemet-css
 ```
-npm install @digdir/design-system-react @digdir/design-system-tokens
+
+#### Typescript
+
+If you use Typescript, make sure you have typescript >= 3.8 as `devDependencies`:
+
+```sh
+npm i typescript --save-dev
 ```
 
-### 2. Add the Inter font
+### 2. Font
+
+You are free to use any font-family with the components.
+
+The components are designed and developed using the [Inter font](https://github.com/rsms/inter) so variantions might occur if a different font is used.
+
+#### Add the Inter font (optional)
 
 Add the `<link>` tag to your application and set the `font-family` to `Inter`.
```

Comment:
Add the `<link>` tag in `<head>`, and set the `font-family` to `Inter` in your global css file.

Reply:
Changed to 
> Add the `<link>` tag in `<head>`, and set `font-family` to `Inter` in your global css file.